MetaMask/metamask-extension
# Adding NFT autodetection to "What's New" Announcements

## Explanation
Adding NFT autodetection announcement to "What's new"
* Fixes #17637 



## Screenshots/Screencaps



### Before



### After
<img width="1285" alt="image" src="https://user-images.githubusercontent.com/10986371/219428076-a8d3c93a-e810-481b-a759-cf6116a2557a.png">



## Manual Testing Steps
- Install a MetaMask extension build from this branch 
- Install dependencies with `yarn`
- Run with: `NFTS_V1=1 COLLECTIBLES=1 COLLECTABLES=1 yarn start`
- Upon opening the app, a new announcement should be shown



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
